kits/data-health-inspector/README.md

```
@@ -0,0 +1,18 @@
+# Data Health Inspector
+
+This Lamatic AgentKit template provides an automated workflow to inspect datasets for quality issues, structural inconsistencies, and anomalies.
+
+## What it does
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)